ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/stack/StackComponentView.kt

```diff
@@ -497,16 +497,16 @@ private fun MainStackComponent(
         }
     }
 
-    val backgroundColorStyle = stackState.backgroundColor?.forCurrentTheme
+    val backgroundStyle = stackState.background?.let { rememberBackgroundStyle(background = it) }
```

Comment:
When testing the `Background` modifier that accepts a `BackgroundStyle` in these new components, it seems that there are some sizing issues until the image has downloaded from web, causing some weird reloading. I mean to test it more thoroughly in case we can do something to the background modifier... Otherwise, we might need to treat the image background differently in order to avoid the resizing that's currently happening...

Reply:
So the issue with the sizing still is happening on the preview, where the stack takes the whole available space. However, it seems it gets fixed by following the workaround in https://github.com/coil-kt/coil/issues/1910. Fixed in https://github.com/RevenueCat/purchases-android/pull/2131/commits/d71b22426700eea873a5dced478a7946cf7c42cb

Reply:
Another update, looks like that change didn't actually fix the issue, there is still resizing noticeable in slow connections. It's not the end of the world since it only happens the first time we're loading the background images, but still something we should fix.